modules/persistent-storage-csi-migration-vsphere-to-4-13.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-migration.adoc
+
+:_content-type: CONCEPT
+[id="persistent-storage-csi-migration-vsphere-to-4.13_{context}"]
+= Updating from {product-title} 4.12 to 4.13 with vSphere in-tree PVs
+
+If you are using vSphere in-tree persistent volumes (PVs) and want to update from {product-title} 4.12 to 4.13, update vSphere vCenter and ESXI host to 7.0 Update 3L or 8.0 Update 2, otherwise the {product-title} upgrade is blocked. After updating vSphere, your {product-title} update can occur and automatic CSI migration for vSphere occurs only if you opt in.
+
+Alternatively, if you do not want to update vSphere, you can proceed with an {product-title} update by running the following command to perform an administrator acknowledgment:
+
+[source, terminal]
+----
+oc -n openshift-config patch cm admin-acks --patch '{"data":{"ack-4.12-kube-126-vsphere-migration-in-4.14":"true"}}' --type=merge
+----
+
+[IMPORTANT]
+====
+If you do *not* update to {product-title} 4.13.10, or later, and do *not* update vSphere, and then opt in to migration (see _Manually enabling CSI automatic migration_ under _Additional Resource_ below), known issues can occur. https://access.redhat.com/node/7011683[Before opting in to migration, carefully read this knowledge base article].
```

Comment:
Opt it is in 4.13 not 4.12 so 
`see _Manually enabling CSI automatic migration_ under _Additional Resource_ below` is confusing because it says "below". The 4.12 manually enabling migration is meant to enable the tech preview feature.

```suggestion
If you do *not* update to {product-title} 4.13.10, or later, and do *not* update vSphere, and then opt in to migration (see _Manually enabling CSI automatic migration_ under _Additional Resource_ below), known issues can occur. https://access.redhat.com/node/7011683[Before opting in to migration, carefully read this knowledge base article].
```suggestion
If you do *not* update to {product-title} 4.13.10, or later, and do *not* update vSphere, and then opt in to migration (see _Manually enabling CSI automatic migration_ under _Additional Resource_ below), known issues can occur. https://access.redhat.com/node/7011683[Before opting in to migration, carefully read this knowledge base article].
```
```suggestion
If you do *not* update to {product-title} 4.13.10, or later, and do *not* update vSphere, and then opt in to migration (see _Manually enabling CSI automatic migration_ under _Additional Resource_ below), known issues can occur. https://access.redhat.com/node/7011683[Before opting in to migration, carefully read this knowledge base article].
```dd
```suggestion
If you do *not* update to {product-title} 4.13.10, or later, and do *not* update vSphere, and then opt in to migration (see _Manually enabling CSI automatic migration_ under _Additional Resource_ below), known issues can occur. https://access.redhat.com/node/7011683[Before opting in to migration, carefully read this knowledge base article].
It should point out to the 4.13 opt-in doc instead.

Reply:
"Below" in that sentence refers to the location of the Additional Resources heading. It doesn't refer at all to the version of OCP, but I've removed it to take away any confusion.